rest-api/src/main/java/org/jboss/pnc/rest/api/endpoints/OperationEndpoint.java

```diff
@@ -71,6 +73,7 @@ public interface OperationEndpoint {
     static final String COMPLETE_OPERATION_DESC = "Complete a running operation.";
     static final String OPERATION_RESULT = "Result of completed operation.";
     static final String UPDATE_DEL_ANALYZER_DESC = "Updates an existing deliverable analyzer operation.";
+    static final String START_DELIVERABLE_ANALYSIS = "Starts the new deliverable analysis above artifacts given by URLs";
```

Comment:
```suggestion
    static final String START_DELIVERABLE_ANALYSIS = "Starts a new deliverable analysis of deliverables given by URLs";
```

Reply:
Aaaaah, you should sign off that english course, because you will bully me regarding my javadocs  -_-
[821c8a8](https://github.com/project-ncl/pnc/pull/3825/commits/821c8a806056840136e7ff4ee1bc45c0da6ec60f)